src/Uno.UI/UI/Xaml/Controls/Primitives/ToggleButton.cs

```diff
@@ -43,19 +55,24 @@ public ToggleButton()
 				typeof(ToggleButton),
 				new PropertyMetadata(
 					false,
-					propertyChangedCallback: (s, e) => ((ToggleButton)s).OnIsCheckedChanged(e.OldValue as bool?, e.NewValue as bool?)
-				)
+					propertyChangedCallback: (s, e) => ((ToggleButton)s).OnIsCheckedChanged())
 			);
-		#endregion
 
-		protected virtual void OnIsCheckedChanged(bool? oldValue, bool? newValue)
+		protected virtual void OnIsCheckedChanged()
```

Comment:
Why did you removed the `newValue` ? I would assume that a cast is less expensive than accessing a DP.

BTW this is a breaking change, and actually this method [does not exists on UWP](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.togglebutton), so if you change the signature you should make it `private` or `internal`.

Reply:
It was an alignment that does not need to be there. I'll take note of that API so we can remove it later on.